app/Http/Controllers/VerificationApiController.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Services\User\EmailVerificationService;
+use Illuminate\Http\Request;
+
+/**
+ * @group Email Verification
+ *
+ * APIs for verify email and resend verification email.
+ */
+class VerificationApiController extends Controller
+{
+    protected $emailVerification;
+
+    public function __construct(EmailVerificationService $emailVerification)
+    {
+        $this->emailVerification = $emailVerification;
+    }
+
+    /**
+     * Verify Email
+     *
+     * @urlParam user_id int required The ID of the user.
+     * @queryParam expires string required expire datetime for this link.
+     * @queryParam hash string required Generated from user data.
+     * @queryParam signature string required Uniquely identifier for this link.
+     *
+     */
+    public function verify($user_id, Request $request)
+    {
+        $response = $this->emailVerification->executeVerify($user_id, $request);
+        return view('email.verified', ["msg" => $response['msg'], "allowResend" => $response['allowResend']]);
```

Comment:
This is supposed to be an API, why is there a view returned? All frontend pages are supposed to be handled in React, please, stick to that.

Reply:
There is no need to have front end page for such feature, it's just a page to confirm that they verified their emails successfully, nothing more.

Reply:
We agreed that the backend would be strictly for APIs. We can't have the user be clicking 2 different domains no matter how simple the page is. A success or error response should be returned and since it's a simple page, the same exact page could be easily implemented in React.